_episodes/12-live.md

```diff
@@ -14,6 +14,13 @@ keypoints:
 - "Live coding allows instructors to follow learners."
 - "The mistakes are the pedagogy."
 ---
+
+One of the cornerstones of Software and Data Carpentry teaching is live coding.  This 
```

Comment:
While you are at it, could we clean this part up a bit? I.e, this part becomes

>One of the cornerstones of Software and Data Carpentry teaching is live coding.: instructors don't use slides, but work through the lesson material, typing in the code or instructions, with the workshop participants following along. This section explains how it works, why we use it, and gives general tips for an effective live coding presentation.

Then change the sentence further down that starts with "One of the key teaching practices used at Software and Data Carpentry workshops is live coding" to "Below follow ten tips to help you get started:"